### PR TITLE
Filter only accepted processes in Process comboBox

### PR DESCRIPTION
**Pull request by Demnokkoyen**
This PR implements what is described in #96.

**Before this PR is merged:**
It is required to update https://raw.githubusercontent.com/4RTools/4RTools/main/supported_servers.json to add a property that's the game process's name.

### CODE REVIEW

**Review by afonsocdaniel — APPROVED** [review]
LGTM 🎉

**Comment by markiing** [comment]
hey @Demnokkoyen 

`ClientListSingleton.ExistsByName(p.ProcessName);` 

Don't need to put this into a variable, can call directly in if :)

File: Forms/Container.cs
Comment (markiing) [inline Forms/Container.cs:209]: Call directly `ClientListSingleton.ExistsByName(p.ProcessName)`

Comment (afonsocdaniel) [reply Forms/Container.cs:209]: @Demnokkoyen just a small suggestion, once you address this, we can merge the PR. Thanks for the contribution! 👍

Comment (Demnokkoyen) [reply Forms/Container.cs]: @afonsocdaniel fixed it! Just a small suggestion: DON'T merge this PR before setting every processName property in `supported_servers.json`. Servers without the property set to a valid value won't work be shown inside the process selectbox; I'm asking for your help cause I don't have all of these servers installed on my machine.

Comment (markiing) [reply Forms/Container.cs]: You don't need this extra property. You can use execName that already exists in Client.

File: Model/Hexed.cs
Comment (markiing) [inline Model/Hexed.cs:72]: You don't need this extra property `processName` . Exec name is exactly you want :D

File: supported_servers.json
Comment (markiing) [inline supported_servers.json:7]: `processName` is unecessary.

**Comment by markiing** [comment]
LGTM 🎉

**Pull request merged**
